src/main/scala/tile/Core.scala

```diff
@@ -3,7 +3,7 @@
 package freechips.rocketchip.tile
 
 import Chisel._
-
+import Chisel.ImplicitConversions._
```

Comment:
I don't actually understand how this provides `.W`

Reply:
Via indirection, this is just https://github.com/freechipsproject/chisel3/blob/387274784115bee2bf7167547a7ee459219e1413/src/main/scala/chisel3/util/ImplicitConversions.scala#L12 which provides implicit conversion from `Int` to `UInt` and `Boolean` to `Bool`.

Reply:
ok, so I should just get rid of this import?